sky/cli.py

```diff
@@ -3063,24 +3063,29 @@ def _output():
                 return
 
         # Show detailed accelerator information
-        accelerator_split = accelerator_str.split(':')
-        if len(accelerator_split) > 2:
-            raise click.UsageError(
-                f'Invalid accelerator string {accelerator_str}. '
-                'Expected format: <accelerator_name>[:<quantity>].')
-        if len(accelerator_split) == 2:
-            name = accelerator_split[0]
-            # Check if quantity is valid
-            try:
-                quantity = int(accelerator_split[1])
-                if quantity <= 0:
-                    raise ValueError('Quantity cannot be non-positive integer.')
-            except ValueError as invalid_quantity:
-                raise click.UsageError(
-                    f'Invalid accelerator quantity {accelerator_split[1]}. '
-                    'Expected a positive integer.') from invalid_quantity
+        if accelerator_str is None:
```

Comment:
This case has been handled above by 
https://github.com/skypilot-org/skypilot/pull/1975/files#diff-c0e074dfacd8c54c331ba25496731d06820e0f93eab3673840879bc1d445c1bbR3033? Or did I miss anything?

Reply:
Ah thanks - I've handled it in the same if condition now. Tested again.